lumigator/python/mzai/backend/backend/api/routes/jobs.py

```diff
@@ -194,7 +240,7 @@ def get_job_result_download(
     return service.get_job_result_download(job_id)
 
 
-def _get_all_ray_jobs() -> list[JobSubmissionResponse]:
+def _get_all_ray_jobs() -> list[RayJobDetails]:
```

Comment:
why are we changing this?

Reply:
Because the info we get is modelled by the Ray pydantic models. _Then_ it is merged with our own job info.

Reply:
The merged information is not modelled as `JointJobInfo(RayJobDetails, JobResponse)` because this would require the lumigator schemas to import the `ray` package, which is too heavy for the end user as there is no separate api or model package for Ray.